docs/v3/automate/add-schedules.mdx

```diff
@@ -268,6 +289,72 @@ A 9 AM daily schedule with a DST-aware start date maintains a local 9 AM time th
 with a UTC start date maintains a 9 AM UTC time.
 </Note>
 
+## Associating parameters with schedules
+Using any of the above methods to create a schedule, you can bind parameters to your schedules.
+
+For example, say you have a flow that sends an email. Every day at 8:00 AM you want to send a message to one recipient, and at 8:05 AM you want to send a different message to another recipient.
+
+Instead of creating independent deployments with different default parameters and schedules, you can bind parameters to the schedules themselves:
+
+### Schedule parameters in Python
+
+Whether using `.serve` or `.deploy`, you can pass `parameters` to your deployment `schedules`:
+
+```python send_email_flow.py {13,17-20}
+from prefect import flow, parameter
+from prefect.schedules import Cron
+
+@flow
+def send_email(to: str, message: str = "Stop goofing off!"):
+    print(f"Sending email to {to} with message: {message}")
+
+send_email.serve(
+  name="my-flow",
+  schedules=[
+    Cron(
+      cron="0 8 * * *",
+      parameters={"to": "jim.halpert@dundermiflin.com"}
+    ),
+    Cron(
+      cron="5 8 * * *",
+      parameters={
+        "to": "dwight.schrute@dundermiflin.com",
+        "message": "Stop goofing off! You're assistant _to_ the regional manager!"
+      }
+    )
+  ]
+)
+```
+Note our flow has a default `message` parameter, but we've overridden it for the second schedule.
+
+This deployment will schedule runs that:
+
+- Send "Stop goofing off!" to Jim at 8:00 AM every day
+- Send "Stop goofing off! You're assistant _to_ the regional manager!" to Dwight at 8:05 AM every day
+
+<Tip>
+Use the same pattern for any schedule type in `prefect.schedules`, e.g. `Interval` and `RRule`.
+</Tip>
```

Comment:
I think you should expand this tip to mention that the same thing works even if using a single schedule and the `schedule=` kwarg